ballista/core/src/execution_plans/coalesce_tasks.rs

```diff
@@ -174,9 +164,7 @@ impl ExecutionPlan for CoalesceTasksExec {
                         return;
                     }
                 }
-            };
-
-            join_handles.push(tokio::spawn(fut.in_current_span()));
```

Comment:
This should not have been reverted. It is needed for tracing propagation. Please unrevert.